tests/integration/test_hyper_transformer.py

```diff
@@ -1353,8 +1379,9 @@ def test_random_seed(self):
 
         # Run
         ht.fit(data)
-        transformed = ht.transform(data)
-        reversed1 = ht.reverse_transform(transformed)
+        ht.reset_randomization()
```

Comment:
I think that this should be solved before going any further, this should be called after `fit` on the `hypertransformer` as per #716